chrome/locale/en-US/zotero/zotero.ftl

```diff
@@ -219,11 +219,11 @@ rtfScan-introPage-description2 = To get started, select an RTF input file and an
 rtfScan-input-file = Input File
 rtfScan-output-file = Output File
 
-zotero-file-none-selected =
-    .value = No file selected
-
-zotero-file-choose =
-    .label = Choose File…
+rtfScan-no-file-selected = No file selected
+rtfScan-choose-input-file =
+    .label = Choose Input File...
+rtfScan-choose-output-file =
+    .label = Choose Output File...
```

Comment:
1. Real ellipses: `...` → `…`
2. Can we use `aria-label` for these and keep `label` as "Choose File…"? Visually, this is a bit overly verbose, and the buttons aren't aligned properly with labels.

Reply:
> Real ellipses: ... → …

Oh, I did not actually know there is a separate character for ellipses... Good to know!

> Can we use aria-label for these and keep label as "Choose File…"? Visually, this is redundant, and the buttons aren't aligned properly with different labels.

I think so... My understanding is that in general, it's best to be consistent across the board. But the VPAT issue did specifically refer to the "accessible name" not being descriptive, which I assume is what is being announced by screen reader. So we can just go with the aria-label for now and tweak it later if they get back with more comments

Reply:
Well, the other option would be to link each button to each label ("Input File", "Output File"), I guess with `aria-describedby` (?), and then avoid an extra localized string. But I don't know how that would be read.

Reply:
`aria-describedby` would sound a bit off, I think. It's generally meant for context that is announced after the labels and main properties of the item, sometimes after a pause. So, for instance, with voiceover, it would say "Choose file, button, has focus", pause, "Input file", which sounds off.

We can consider `aria-labelledby` that would point at the button itself and the label. In that case, I am hearing "Choose file... Input file" as one announcement. It gets the message across though it does sound a bit clunky with two "file" strings. But to avoid multiple localized strings, perhaps it's worth it?